public/js/reducers/breakpoints.js

```diff
@@ -123,6 +123,9 @@ function update(state = State(), action: Action) {
       }
 
       break;
+    }
+    case "NAVIGATE": {
+      return State();
```

Comment:
> I think this is actually a question of using a selector that only queries for breakpoints at that domain or breakpoints that have a source in the store, which can be difficult.

Oh, that's a really good point. I forgot that the breakpoints list is going to show all of them. We probably should restrict `getBreakpoints` to filter out ones that are only associated with current sources. Hm, I hadn't really thought about that, there might be a more elegant 

> I think this is actually a question of using a selector that only queries for breakpoints at that domain or breakpoints that have a source in the store, which can be difficult.

Oh, that's a really good point, I forgot that the breakpoints list is going to show all of them. `getBreakpoints` should definitely only return breakpoints for the current page... I like the idea of checking if the source exists. Not sure if there's a more elegant way to handle this...